Packs/PrismaCloud/Integrations/RedLock/RedLock.py

```diff
@@ -375,6 +375,53 @@ def translate_severity(alert):
         return 1
     return 0
 
+def get_RQL_response():
+
+    """"
+    Retrieve any RQL
+    """
+    RQL = demisto.getArg('rql').encode("utf-8")
+
+    payload = {"query":RQL, "filter": {}}
+
+    handle_filters(payload['filter'])
+    handle_time_filter(payload['filter'], {'type': 'to_now', 'value': 'epoch'})
+
+    if not RQL:
+        return_error('You must specify an RQL to retrieve the response')
```

Comment:
no need for this check, RQL arg should be mandatory for this command. you can see it is required in the yml

Reply:
ahh that's a good point.  removed these lines.